scroll-animations-1/Overview.bs

```diff
@@ -732,6 +732,9 @@ spec: cssom-view-1; type: dfn;
 	If the timeline does not have a corresponding [=named timeline range=],
 	then any keyframes attached to points on that [=named timeline range=]
 	are ignored.
+	If the attachment points are outside the effective range of the animation
+	(e.g. due to 'animation-delay'),
+	the first/last keyframes of the animation are found by interpolation.
```

Comment:
I don't think there's any change needed here, we should use the specified keyframes as is.

What needs to change is [the text in css-animations-1](https://www.w3.org/TR/css-animations-1/#keyframes) where it says:
> If a 0% or from keyframe is not specified, then the user agent constructs a 0% keyframe using the computed values of the properties being animated. If a 100% or to keyframe is not specified, then the user agent constructs a 100% keyframe using the computed values of the properties being animated.

Instead, this needs to only construct 0% or 100% keyframes for properties which don't have keyframes < 0% or greater than 100%.

Reply:
OK, tried rephrasing